src/worker/app.ts

```diff
@@ -2,21 +2,31 @@
 //        instance, mainly to use the `auth()` method. Probot needs refactored
 //        so that method is more easily accessible.
 
-import { createProbot } from 'probot';
+import {Application, createProbot} from 'probot';
 import { findPrivateKey } from 'probot/lib/private-key';
+import setupDeepcheck from "../deepcheck";
 
-const probot = createProbot({
+export const probot = createProbot({
   id: Number(process.env.APP_ID),
   cert: findPrivateKey(),
 
   // These aren't needed by worker process
   secret: undefined,
-  port: undefined,
+  port: Number(process.env.WORKER_PORT) || 8080,
```

Comment:
port 8080 would interfere with the main API port.

Reply:
No it wouldn't, when running in Micros as a "worker", there will be no main "API". When we run the main version, this Probot wouldn't start.

When we run locally the "WORKER_PORT" env variable will be used, which will be different from the main API port

Reply:
I'm talking running it locally with `npm start`.  It should work out of the box without the need for the env var.

Reply:
Our process of running things locally involves copying .env.example file to .env and stting up env variables. So we should be good

Reply:
I know we can be, but I'm saying that environment variables, by nature, are optional.  You are setting up a potential code path that will make the port conflict with each other.  All you need to do is set the default port for the worker to 8081 and it won't be a problem.